modules/luci-mod-network/htdocs/luci-static/resources/view/network/network.js

```diff
@@ -113,7 +113,7 @@ L.poll(5, L.url('admin/network/iface_status', networks.join(',')), null,
 					var e = document.getElementById(ifc.id + '-ifc-edit');
 					if (e) e.disabled = true;
 
-					var link = L.url('admin/system/packages') + '?query=luci-proto&display=available';
+					var link = L.url('admin/system/opkg');
```

Comment:
The query parameter still works and should be kept imho.

Reply:
@jow- , Thank you, I just checked the code and it's indeed the case ;)